src/coreclr/tools/aot/ILLink.Shared/DiagnosticId.cs

```diff
@@ -28,7 +28,7 @@ public enum DiagnosticId
 		UnrecognizedCommandLineOption = 1015,
 		InvalidWarningVersion = 1016,
 		InvalidGenerateWarningSuppressionsValue = 1017,
-		MissingArgumentForCommanLineOptionName = 1018,
+		MissingArgumentForCommandLineOptionName = 1018,
```

Comment:
This introduces more diffs with the other DiagnosticId. I don't know if it's an improvement. I would do this kind of things once we don't have three actively maintained copies of this file.